lib/tasks/install-blueprint.js

```diff
@@ -100,21 +101,37 @@ class InstallBlueprintTask extends Task {
 
   _gitClone(source, destination) {
     logger.info(`Cloning from git (${source}) into "${destination}" ...`);
-    return execa('git', ['clone', source, destination]);
+    return this.execa('git', ['clone', source, destination]);
   }
 
-  _npmInstall(cwd) {
-    logger.info(`Running "npm install" in "${cwd}" ...`);
+  _npmInstall(cwd, options) {
+    let command = 'npm';
+    let args = ['install'];
+
+    if (options && options.yarn === true) {
+      command = 'yarn';
+      args = ['install'];
+    }
+
+    logger.info(`Running "${command} ${args.join(' ')}" in "${cwd}" ...`);
 
     this._copyNpmrc(cwd);
-    return execa('npm', ['install'], { cwd });
+    return this.execa(command, args, { cwd });
   }
 
-  _npmInstallModule(module, cwd) {
-    logger.info(`Running "npm install ${module}" in "${cwd}" ...`);
+  _npmInstallModule(module, cwd, options) {
+    let command = 'npm';
+    let args = ['install', module];
+
+    if (options && options.yarn === true) {
+      command = 'yarn';
+      args = ['add', module];
```

Comment:
the problem with this might be that it will add the module to the `package.json`, which might not be intended. npm@5 does the same though, so this might be a problem in any case now 🤔